quarkus-cli/pom.xml

```diff
@@ -27,8 +27,26 @@
                 </property>
             </activation>
             <properties>
+                <!-- To not build the module on Native -->
                 <quarkus.package.type>fast-jar</quarkus.package.type>
             </properties>
+            <!-- Skip on Native as JVM run will cover the Native build too -->
```

Comment:
This can be a bit problem for Jenkins jobs when I expect to have JVM and Native variants of the jobs.

This way we would need to specify Mandrel image for JVM job too ?
And there would be no cli tests executed on NATIVE job?

Reply:
No CLI tests executed on Native job.
But good point about the Mandrel image.. we're losing this coverage